components/feature/session/src/main/java/mozilla/components/feature/session/EngineViewPresenter.kt

```diff
@@ -20,8 +20,7 @@ class EngineViewPresenter(
      * Start presenter and display data in view.
      */
     fun start() {
-        renderSession(sessionManager.selectedSession)
-
+        sessionManager.all.forEach { handleNewSession(it) }
```

Comment:
I do not understand why we loop over all sessions here and then later render it if it's a custom tab or selected. Isn't this the same as the previous code that just rendered the selected session? With that we may render multiple things if there are multiple sessions (selected + multiple custom tabs) and just settle on the last one.

I think the underlying problem you are trying to solve is that we never display a custom tab right now because it is never the selected one(*) [at least if there's already one session]?

Essentially Custom Tabs and the browser will live in different activities. So I guess what we will need to do is to tell renderView to either render "whatever is selected" or a specific (custom tab) session. That's what we essentially do in Focus with BrowserFragment.

All this makes me think whether we want to delay displaying custom tabs in the sample browser until we have made the separation at the activity level.

(*) We actually should make sure that a custom tab doesn't get selected because this doesn't make any sense. But that's something for another issue.